app/src/main/java/com/bleyl/recurrence/database/DatabaseHelper.java

```diff
@@ -119,6 +123,13 @@ public void onUpgrade(SQLiteDatabase database, int oldVersion, int newVersion) {
                     + COL_FRIDAY + " TEXT, "
                     + COL_SATURDAY + " TEXT) ");
         }
+
+        if (oldVersion < 4) {
+            database.execSQL("ALTER TABLE " + NOTIFICATION_TABLE + " ADD " + COL_NAG_TIMER + " INTEGER");
+            database.execSQL("ALTER TABLE " + NOTIFICATION_TABLE + " ADD " + COL_ACTIVE_STATE + " BOOLEAN");
+            database.execSQL("UPDATE " + NOTIFICATION_TABLE + " SET " + COL_NAG_TIMER + " = 0;");
+            database.execSQL("UPDATE " + NOTIFICATION_TABLE + " SET " + COL_ACTIVE_STATE + " = FALSE;");
```

Comment:
I didn't know booleans weren't supported in SQLite, I changed all into integers but now I get another error when overwriting an older install:

FATAL EXCEPTION: main
                                                                    Process: com.bleyl.recurrence, PID: 4301
                                                                    java.lang.IllegalArgumentException: Unknown color
                                                                        at android.graphics.Color.parseColor(Color.java:235)
                                                                        at com.bleyl.recurrence.adapters.ReminderAdapter.onBindViewHolder(ReminderAdapter.java:85)
                                                                        at com.bleyl.recurrence.adapters.ReminderAdapter.onBindViewHolder(ReminderAdapter.java:33)
                                                                        at android.support.v7.widget.RecyclerView$Adapter.onBindViewHolder(RecyclerView.java:5217)
                                                                        at android.support.v7.widget.RecyclerView$Adapter.bindViewHolder(RecyclerView.java:5250)
                                                                        at android.support.v7.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:4487)
                                                                        at android.support.v7.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:4363)
                                                                        at android.support.v7.widget.LinearLayoutManager$LayoutState.next(LinearLayoutManager.java:1961)
                                                                        at android.support.v7.widget.LinearLayoutManager.layoutChunk(LinearLayoutManager.java:1370)
                                                                        at android.support.v7.widget.LinearLayoutManager.fill(LinearLayoutManager.java:1333)
                                                                        at android.support.v7.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:562)
                                                                        at android.support.v7.widget.RecyclerView.dispatchLayout(RecyclerView.java:2900)
...
                                                                        at android.support.v7.widget.RecyclerView.onLayout(RecyclerView.java:3071)
                                                                        at android.view.View.layout(View.java:16630)
                                                                        at